components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -7713,6 +7709,29 @@ private boolean deleteComment(String uuid, String commentId, Connection connecti
         return false;
     }
 
+    private void deleteAPIComments(String uuid, Connection connection) throws APIManagementException {
+        int id = -1;
+        ResultSet resultSet = null;
+        try {
+            id = getAPIID(uuid, connection);
```

Comment:
no need to retrieve id again since we already have id in here
https://github.com/wso2/carbon-apimgt/pull/11021/files#diff-ac8fc1b7b2d10089084f9f842ad03ac096b64f8c555798aa18a926e0ec663a4cL6825